CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Improve highlighting of type parameters and `module type of` (#461)
+- Detect Opam and Esy sandboxes (#445)
```

Comment:
Since this a big feature, it's worth expanding on this a bit. E.g.

> Automatic detection of Opam and Esy sandboxes. Opam sandboxes are detected via local switches, and esy via esy.json files.